tests/integration_tests/functional/test_aarch64_armshaker.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Opcode fuzzer test for aarch64."""
+
+import platform
+import time
+
+import pytest
+
+from framework import utils
+from framework.utils_cpu_templates import nonci_on_arm
+
+PLATFORM = platform.machine()
+
+TIMEOUT = 4 * 60 * 60
+
+
+def run_armshaker(vm):
+    """
+    Run armshaker inside uVM
+    """
+
+    # Git clone armshaker
+    # We are using fork of `armsharker` because original version
+    # does not compile with new versions of `glibc`
+    # (`SIGSTKSZ` is not a constant anymore)
+    cmd = "git clone https://github.com/ShadowCurse/armshaker"
```

Comment:
if we plan to have it then I think the fork should be under firecracker-microvm and not under personal forks.